node/index.d.ts

```diff
@@ -1891,8 +1891,27 @@ declare module "dns" {
         priority: number
     }
 
-    export function lookup(domain: string, family: number, callback: (err: Error, address: string, family: number) => void): string;
-    export function lookup(domain: string, callback: (err: Error, address: string, family: number) => void): string;
+    // Supported getaddrinfo flags
+    export const ADDRCONFIG: number;
+    export const V4MAPPED: number;
+
+    export interface LookupOptions {
```

Comment:
Good catch. Interface `LookupOptions` contains only optional properties. It is the same as empty interface `{}`. So, I don't known how it should be fixed.